src/database/Database.cpp

```diff
@@ -346,18 +334,17 @@ Database::addTextColumn(std::string const& table, std::string const& column)
 }
 
 void
-Database::addTextColumnWithDefault(std::string const& table,
-                                   std::string const& column,
-                                   std::string const& defaultVal)
+Database::addTextColumnAndFill(std::string const& table,
```

Comment:
Let's remove this if it isn't used anymore. I don't believe in maintaining code that we aren't using, because it requires effort for no reward. Furthermore, I believe heavily in testing and if code is unused then it is _definitely_ untested.

Reply:
It was tested quite a bit while it _was_ being used, and anyone who were to use it in the future certainly ought to add a test for that use.  :)  But I'll remove it.